sound_play/src/sound_play/flite_plugin.py

```diff
@@ -14,22 +14,26 @@ class FlitePlugin(SoundPlayPlugin):
 
     def __init__(self):
         super(FlitePlugin, self).__init__()
-        self.rospack = rospkg.RosPack()
-        self.default_voice_path = os.path.join(
-            self.rospack.get_path('sound_play'),
-            'resources/flitevox')
+        self._default_voice_path = None
+
+    def get_default_voice_path(self):
+        if self._default_voice_path is None:
+            self._default_voice_path = os.path.join(
+                rospkg.RosPack().get_path('sound_play'),
+                'resources/flitevox')
+        return self._default_voice_path
 
     def sound_play_say_plugin(self, text, voice):
         if voice is None or voice == '':
-            voice = self._default_voice
```

Comment:
`_default_voice` is set here.
https://github.com/ros-drivers/audio_common/blob/c0a6bd3dba08307b3c5ff82eed1498879849b718/sound_play/src/sound_play/flite_plugin.py#L13

we don't need to change this line, I think.
Other parts look good to me.

Reply:
Thanks, I overlooked it actually. Pruned the commit.